sql/moz-fx-data-shared-prod/firefox_accounts_derived/health_tags_v1/schema.yaml

```
@@ -0,0 +1,22 @@
+fields:
+  - mode: NULLABLE
+    name: timestamp
+    type: TIMESTAMP
+  - mode: NULLABLE
+    name: server
+    type: STRING
+  - mode: NULLABLE
+    name: version
+    type: STRING
+  - mode: NULLABLE
+    name: commit
+    type: STRING
+  - mode: NULLABLE
+    name: l10n
+    type: STRING
+  - mode: NULLABLE
+    name: tospp
+    type: STRING
+  - mode: NULLABLE
+    name: since_last_push
+    type: STRING
```

It seems like the `since_last_push` column contains float values.
```suggestion
    type: FLOAT
```

---

done